### PR TITLE
#1276 Use unit test job instead of coverage

### PR DESCRIPTION
**Pull request by lecourtoisflo**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,62 +34,64 @@ jobs:
...
+          $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests
+
   #     - name: Sonarcloud
   #       working-directory: ${{ runner.workspace }}/dynawo
   #       env:
   #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   #         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
   #         CONTAINER_WORKSPACE: ${{ runner.workspace }}
-  #       run: $CONTAINER_WORKSPACE/sonar/bin/sonar-scanner -Dsonar.host.url=https://sonarcloud.io -Dsonar.cfamily.threads=2
+  #       run: $CONTAINER_WORKSPACE/s
```
Comment (gautierbureau) [inline .github/workflows/ci.yml:97]: wrong delete ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,62 +34,64 @@ jobs:
...
+      # In a PR use pre-built 3rd parties to optimize the duration of the job
+      # For now, run only tests, not coverage
+      - name: Build branch PR
+        env:
+          CONTAINER_WORKSPACE: ${{ runner.workspace }}
+          DYNAWO_THIRD_PARTY_INSTALL_DIR: /opt/dynawo/3rParty
+        if: github.ref != 'refs/heads/master' && github.event_name == 'push'
+        run: |
+          export DYNAWO_HOME=$(pwd)
+          $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests
```
Comment (gautierbureau) [inline .github/workflows/ci.yml:79]: comment line and use only util/envDynawo.sh build-tests, wrapper is for sonar

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -34,62 +34,64 @@ jobs:
...
+
+      # For now, run only tests, not coverage
+      - name: Build master
+        env:
+          CONTAINER_WORKSPACE: ${{ runner.workspace }}
+        if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+        run: |
+          export DYNAWO_HOME=$(pwd)
+          util/envDynawo.sh build-3rd-party
+          $CONTAINER_WORKSPACE/build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir bw-output util/envDynawo.sh build-tests
```
Comment (gautierbureau) [inline .github/workflows/ci.yml:89]: same here

**Review by gautierbureau — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lecourtoisflo) [reply .github/workflows/ci.yml:97]: OK

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lecourtoisflo) [reply .github/workflows/ci.yml:79]: OK

File: .github/workflows/ci.yml (hunk 3 above)
Comment (lecourtoisflo) [reply .github/workflows/ci.yml:89]: OK

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**